unsuspend.lua

```diff
@@ -184,9 +185,20 @@ if dfhack_flags.module then
     return
 end
 
-local unsuspended_count, flow_count, buildingplan_count = 0, 0, 0
+local opts = utils.invert{...}
+local quiet = opts['-q'] or opts['--quiet']
+local smart = opts['-s'] or opts['--smart']
```

Comment:
this is *ok* for now, but at some point you should just use a real commandline parsing function like `argparse.processArgsGetopt()` and avoid having to do the matching and parsing yourself. docs here: https://docs.dfhack.org/en/latest/docs/dev/Lua%20API.html#argparse